portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp

```diff
@@ -143,7 +152,14 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 					%>
 
 						<li class="page-item <%= (i == cur) ? "active" : StringPool.BLANK %>">
-							<a class="page-link" href="<%= _getHREF(formName, namespace + curParam, i, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, i) : "" %>"><span class="sr-only"><liferay-ui:message key="page" /></span><%= i %></a>
+							<c:if test="<%= i == cur %>">
+								<a aria-current="page" class="page-link">
```

Comment:
Very interesting!

When taking a look on https://www.aditus.io/aria/aria-current/, this a11y issue can be reproduced in breadcrumbs navigation and date pickers.